clouddriver-ecs/src/test/groovy/com/netflix/spinnaker/clouddriver/ecs/TestCredential.groovy

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017 Lookout, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.ecs
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.clouddriver.aws.security.NetflixAmazonCredentials
+import groovy.transform.CompileStatic
+
+@CompileStatic
+class TestCredential {
```

Comment:
I'm not quite sure why this class is in this PR.  Why? :)

Reply:
`accountCredentialsProvider.getAll() >> [TestCredential.named('')]`

For testing. Time-to-time I need that class for testing, the AWS folks didn't make the class public, so I just copied it over to our module.